src/main/java/com/github/tomakehurst/wiremock/http/RequestMethod.java

```diff
@@ -46,29 +46,20 @@ public static RequestMethod fromString(String value) {
     return new RequestMethod(value);
   }
 
+  @JsonCreator
+  public static Set<RequestMethod> fromSet(Set<String> values) {
+    return values.stream().map(RequestMethod::fromString).collect(Collectors.toSet());
+  }
+
   @JsonValue
   public String value() {
     return name;
   }
 
-  public boolean isOneOf(RequestMethod... methods) {
```

Comment:
Now it is a breaking change, right?

Reply:
@oleg-nenashev 
Hello
I think RequestMethodMatcher and its use in RequestPattern and Diff are more suitable

Reply:
Yes, I do not disagree. The problem is the API breakage. We cannot track down the usages, so it would ne nice to keep historical methods though it would be perfectly fine to deprecate them and the point users to as proper usage pattern in the annotation

Reply:
My IDE tells me that the method is not currently in use.
![image](https://github.com/wiremock/wiremock/assets/33098392/af81a686-0dda-451b-9c51-a0409b1e6553)
So I don't know what to say...